sdk/core/Azure.Core/tests/NextLinkOperationImplementationTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Core.Tests
+{
+    public class NextLinkOperationImplementationTests
```

Comment:
I'd like to see negative tests where you prove that parameters are being validated and that the behavior when you can't create a URL works as expected.

Reply:
[ThrowOnInvalidUri](https://github.com/Azure/azure-sdk-for-net/pull/42686/files#diff-c80a98a8165ad849356827df7805e5809b17c08b7a411fc12d49fc64dbe5e155R77-R80) and [CreateWithIncompleteNextRequestUri](https://github.com/Azure/azure-sdk-for-net/pull/42686/files#diff-c80a98a8165ad849356827df7805e5809b17c08b7a411fc12d49fc64dbe5e155R15-R30) are negative cases